### DISCUSSION TITLE
Brakes sticking

### DISCUSSION THREAD

**Discussion by winkerbie** [flybywiresim/aircraft#6417] [Help]
Hi, I have limited buttons on my joystick (Logitech 3D Pro). So I have bound a button (not axis) for the brakes. 

However, I find that pressing the button for more than a few seconds causes it to become stuck at 100% - almost always causing my taxing to come to a complete stop.

Does anyone else have this odd behavior. It makes taxing very difficult

**Comment by jcc1953** [comment]
Yes, it's the same for me. It's a probleme (or not) of delay between the moment where you push the button brake and the activation of he hydraulique brakes.
